Json.More/ValueWriter.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text.Json;
+
+namespace Json.More;
+
+internal static class ValueWriter
+{
+#pragma warning disable IDE1006
+	// ReSharper disable InconsistentNaming
+	public static void WriteValues(Utf8JsonWriter writer, IEnumerable<(object?, Type)> values, JsonSerializerOptions options)
+		// ReSharper restore InconsistentNaming
+#pragma warning restore IDE1006
+	{
+		foreach (var item in values)
+		{
+			JsonSerializer.Serialize(writer, item.Item1, item.Item2, options);
+		}
+	}
+
+	public static IEnumerable<(object?, Type)> Unwrap1<T>(ValueTuple<T> value)
```

Comment:
All tuple types implement the `ITuple` interface which provides an IList-like abstraction for accessing elements (and also handle `TRest` values correctly).

Reply:
`ITuple` doesn't exist in .Net Standard.  I looked at that.

Reply:
Also, the ITuple interface returns `object`, but I wouldn't I need the type param to be sure I'm serializing the right type?  I could use the non-generic version of `JsonSerializer.Serialize()`, but that would just serialize what the _object_ is rather than what the _type parameter_ is, and they might be different.